### DISCUSSION TITLE
gcc optimization

### DISCUSSION THREAD

**Discussion by heinono1** [idaholab/moose#24937] [Q&A Getting Started] [answered]
Hi,

Sorry for asking a stupid question, but what is the default optimization flag for gnu9 gcc compiler? Is it -O or -O3 or -Ofast? And what is the reason for the particular default setting?

Thanks,
Olle

**Comment by GiudGiud** [comment, marked as answer]
-O2
The main reason is legacy

It will take some libmesh work to get that upgraded and it's not clear how impactful it would be

**Reply by heinono1** [reply]
OK. Thanks, Guillaume.
